src/View/Sidebar/DeviceListBox.vala

```diff
@@ -173,11 +173,15 @@ public class Sidebar.DeviceListBox : Gtk.Box, Sidebar.SidebarListInterface {
     }
 
     public override void refresh_info () {
-        list_box.get_children ().@foreach ((item) => {
-            if (item is AbstractMountableRow) {
-                ((AbstractMountableRow)item).update_free_space ();
+        int index = 0;
+        Gtk.ListBoxRow? row = list_box.get_row_at_index (index++);
```

Comment:
Same question as another review. It seems odd to me to start at 0 and then immediately increment

Reply:
Now using prefix operator which is clearer.